prophyc/tests/test_prophyc.py

```diff
@@ -700,3 +703,31 @@ def test_model_evaluation_warnings(call_prophyc, tmpdir_cwd):
 prophyc: warning: X::y array has unknown size "UNKNOWN"
 prophyc: error: X byte size unknown
 """
+
+
+def test_python_package_cross_module_imports(call_prophyc, tmpdir_cwd, monkeypatch):
+    codecs_package_dir = tmpdir_cwd.join("some_codecs")
+    codecs_package_dir.mkdir()
+    codecs_package_dir.join("__init__.py").ensure()
+
+    tmpdir_cwd.join("a.prophy").write("""\
+struct A {
+    u32 a;
+};
+""")
+    tmpdir_cwd.join("b.prophy").write("""\
+#include "a.prophy"
+struct B {
+    A a;
+};
+""")
+
+    ret, _, _ = call_prophyc(["--python_out", str(codecs_package_dir),
+                              str(tmpdir_cwd.join("a.prophy")),
+                              str(tmpdir_cwd.join("b.prophy"))])
+    assert ret == 0
+
+    monkeypatch.syspath_prepend(str(tmpdir_cwd))
+
+    from some_codecs import b
+    b.A
```

Comment:
`assert hasattr(b, 'A')` would communicate intent better, you could consider e.g. accessing a field in instantiated object to make sure that sibling module provided necessary definitions: `assert b.B().a.a`